testworld/keystore

```diff
@@ -0,0 +1 @@
+{"address":"89b0a86583c4444acfd71b463e0d3c55ae1412a5","crypto":{"cipher":"aes-128-ctr","ciphertext":"c779f8379d770d92cfc1ddd4a8f31d5a0adc8f2a0b2a1401370d3630f38c0c8a","cipherparams":{"iv":"36c168e73bf980fe75b0727f890a71ad"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"de1be16e3c981944d1eca2b8b27e4e6e0b5bfb43be0376a5d8889fa679a28122"},"mac":"cc128b815555ba1ead7cae9060e8842afca8356d06455bd9a5752ba6fcc092ef"},"id":"45e060a6-d2ae-43b8-922f-44829499d37d","version":3}
```

Comment:
This is for now, next pr will include
- Loading this from a common location within the code, I don't want to have any implicit dependancy on env flags on tests as its hard to figure out whats going on. Although a further discussion is needed to agree on this, I like to remove all env variable dependancies in test suites, it will help us a lot in debugging issues going forward
- Running localgeth node if it hasn't started already
- Loading testworld configs from a actual config json so that we can for example switch networks at will